tests/test_binary.nim

```diff
@@ -271,13 +271,12 @@ proc main =
   const helpStart = &"Usage:\n  {binaryName} [global-options] <command> [command-options]"
 
   const cmd = "nimble --verbose build -d:release"
-  stdout.write(&"Running `{cmd}`... ")
-  stdout.flushFile()
+  stderr.write(&"Running `{cmd}`... ")
   let (buildOutput, buildExitCode) = execCmdEx(cmd, workingDir = repoRootDir)
   if buildExitCode == 0:
-    echo "success"
+    stderr.writeLine "success"
```

Comment:
Shouldn't this be `stdout`?

```suggestion
    stdout.writeLine "success"
```

Reply:
It's the continuation of the `"Running `{cmd}`... ` message, so it depends entirely on that decision.

Reply:
Right.